components/chart/trendlines.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Trendlines
+page_title: Chart Trendlines
```

Comment:
The rest of the articles in this section use the pattern "Chart" - [article topic].
```suggestion
page_title: Chart - Trendlines
```